server/test/com/cloud/network/element/VirtualRouterElementTest.java

```diff
@@ -225,6 +235,62 @@ public void testPrepare() {
 
     }
 
+    @Test
+    public void testGetRouters1(){
```

Comment:
testGetRoutersUnderThisOrTheseCircumstances() ould be a more appropriate name.
testGetRouters1, testGetRouters2 and testGetRouters3 is not really clear.